manim/mobject/vector_field.py

```diff
@@ -536,14 +530,8 @@ def __init__(
         vector_config: Optional[dict] = None,
         **kwargs
     ):
-        self.x_range = x_range or [
-            kwargs.pop("x_min", None) or floor(-config["frame_width"] / 2),
-            kwargs.pop("x_max", None) or ceil(config["frame_width"] / 2),
-        ]
-        self.y_range = y_range or [
-            kwargs.pop("y_min", None) or floor(-config["frame_height"] / 2),
-            kwargs.pop("y_max", None) or ceil(config["frame_height"] / 2),
-        ]
+        self.x_range = x_range
```

Comment:
@icedcoffeeee: Can you check if this is right?

Reply:
```suggestion
        self.x_range = x_range or [
            floor(-config["frame_width"] / 2),
            ceil(config["frame_width"] / 2),
        ]
```
I think it's supposed to look like this